lib/_debugger.js

```diff
@@ -829,13 +829,7 @@ function Interface(stdin, stdout, args) {
   // Run script automatically
   this.pause();
 
-  // XXX Need to figure out why we need this delay
-  setTimeout(function() {
-
-    self.run(function() {
-      self.resume();
-    });
-  }, 10);
+  setImmediate(() => { this.run(() => { this.resume; }); });
```

Comment:
It seems that `.resume()` might be unnecessary:

* [`.run()` calls either `.error()` or `trySpawn()`](https://github.com/nodejs/node/blob/a32f7eb4bec1eb08f271d63f9f99dc322a6e1b6d/lib/_debugger.js#L1036-L1045) 
* [`.error()` will call `.resume()`](https://github.com/nodejs/node/blob/a32f7eb4bec1eb08f271d63f9f99dc322a6e1b6d/lib/_debugger.js#L903)
* [`trySpawn()` will call `.resume()` in a `ready` event listener on `client`](https://github.com/nodejs/node/blob/a32f7eb4bec1eb08f271d63f9f99dc322a6e1b6d/lib/_debugger.js#L1719)